render-test/render_test.cpp

```diff
@@ -194,7 +194,7 @@ int runRenderTests(int argc, char** argv, std::function<void()> testStatus) {
         bool shouldIgnore = false;
         std::string ignoreReason;
 
-        const std::string ignoreName = id;
+        const mbgl::filesystem::path ignoreName(id);
```

Comment:
Could you use `std::filesystem::path` here?

Reply:
The initial change was using `std::filesystem::path` but it didn't compile on iOS CI.
`error: 'path' is unavailable: introduced in iOS 13.0` -> https://github.com/maplibre/maplibre-native/actions/runs/9960886257/job/27521177291#step:9:3136

Reply:
Should work now if you merge in main. :)

Reply:
Changed `mbgl::filesystem` to `std::filesystem`